common/culture/traditions/00_realm_traditions.txt

```diff
@@ -1770,23 +1799,24 @@ tradition_parochialism = {
 		1 = mediterranean
 		4 = city.dds
 	}
-
-	is_shown = { #Replaced by tradition_republican_legacy
-		NOR ={
-			this = culture:roman
-			any_parent_culture_or_above = {
-				this = culture:roman
-			}
-		}
-	}
-	can_pick = {
-		NOR ={
-			this = culture:roman
-			any_parent_culture_or_above = {
-				this = culture:roman
-			}
-		}
-	}
+	
+	# Warcraft
+	# is_shown = { #Replaced by tradition_republican_legacy
+	#	NOR ={
+	#		this = culture:roman
+	#		any_parent_culture_or_above = {
+	#			this = culture:roman
+	#		}
+	#	}
+	# }
+	# can_pick = {
+		# NOR ={
+		#	this = culture:roman
+		#	any_parent_culture_or_above = {
+		#		this = culture:roman
+		#	}
+	#	}
+	# }
```

Comment:
It's prob allowed to have both `tradition_parochialism` and `tradition_republican_legacy` now.

Reply:
`tradition_republican_legacy` (Meritocracy) is now hidden for cultures with `tradition_parochialism` on the basis that they share similar bonuses despite the adapted loc of Meritocracy.

Reply:
I would test it. It may work not like you intented.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/204127879-9d227799-809e-48e1-889d-3592c65268bb.png)

</details>
When you create a new culture, the culture has no tradition untill you click the Create button. So in theory, you can create a culture with both traditions. You pick both, click the Create button.
If my guess is correct, you should use `any_in_list = { list = traits this = tradition_parochialism }` check. This condition should check which traditions you picked in the culture creation screen.

Reply:
Updated with 6d1b0ba11. Tested with diverge and reform; both traditions are exclusive with appropriate localisation.